pkgs/development/python-modules/i3ipc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchurl
+, enum-compat
+, xorgserver, pytest, i3, python
+}:
+
+buildPythonPackage rec {
+  pname = "i3ipc";
+  version = "1.4.0";
+
+  src = fetchurl {
```

Comment:
`fetchFromGitHub`

Reply:
Is it really better? With `fetchurl` one can change only `version` and `sha256` when with `fetchFromGitHub` the `rev` also should be specified. Probably, there is a way to download by tags and not revision?

Reply:
yes, `rev = tag`

Reply:
No, actually you'd have to change `url` and `sha256` with fetchurl. `rev` is definitely easier to change than `url`.

Reply:
Ok, got it and fixed.